std/zig/parser.zig

```diff
@@ -1730,4 +1737,145 @@ test "zig fmt" {
         \\}
         \\
     );
+
+    try testCanonical(
+        \\test "test index" {
+        \\    a[0];
+        \\    a[0 + 5];
+        \\    a[0..];
+        \\    a[0..5];
+        \\}
+        \\
+    );
+
+    try testCanonical(
+        \\test "test array" {
+        \\    const a : [2]u8 = [2]u8{ 1, 2 };
```

Comment:
I think canonical var decl is:
`const name: type = value;`
(No space between colon and var name)